src/instructlab/eval/mt_bench_common.py

```diff
@@ -272,8 +272,7 @@ def chat_completion_openai(
             return response.choices[0].message.content
         except openai.OpenAIError as e:
             if i == API_MAX_RETRY - 1:
-                # Print error on last try
-                print(type(e), e)
+                logger.warning(e)
```

Comment:
maybe even `error`? (@danmcp let me know if there's a good reason to do `info`, as in your snippet.)

Reply:
I was! Will update to `error`.